src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -892,29 +890,63 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
       .lte(runeBalanceCryptoPrecision)
   }, [
     actualRuneWithdrawAmountCryptoPrecision,
-    withdrawType,
+    opportunityType,
     runeAsset,
     runeBalanceCryptoBaseUnit,
     runeDustAmountCryptoBaseUnit,
     runeTxFeeCryptoPrecision,
   ])
 
-  const isBelowMinimumWithdrawAmount = useMemo(() => {
-    const totalWithdrawAmountFiatUserCurrency = bnOrZero(
+  // Protocol fee buffers explained here: https://gitlab.com/thorchain/thornode/-/blob/develop/x/thorchain/querier_quotes.go#L461
+  const minimumWithdrawError = useMemo(() => {
+    if (!poolAsset || !runeAsset) return
+
+    const minimumPoolAssetWithdrawAmountFiatUserCurrency = bnOrZero(slippageFiatUserCurrency).plus(
+      bnOrZero(poolAssetProtocolFeeFiatUserCurrency).times(4),
+    )
+    const minimumRuneWithdrawAmountFiatUserCurrency = bnOrZero(slippageFiatUserCurrency).plus(
+      bnOrZero(runeProtocolFeeFiatUserCurrency),
+    )
+
+    const poolAssetWithrawAmountFiatUserCurrency = bnOrZero(
       actualAssetWithdrawAmountFiatUserCurrency,
-    ).plus(bnOrZero(actualRuneWithdrawAmountFiatUserCurrency))
+    )
+    const runeWithrawAmountFiatUserCurrency = bnOrZero(actualRuneWithdrawAmountFiatUserCurrency)
+
+    if (
+      poolAssetWithrawAmountFiatUserCurrency.gt(0) &&
+      minimumPoolAssetWithdrawAmountFiatUserCurrency.gte(poolAssetWithrawAmountFiatUserCurrency) &&
+      minimumPoolAssetWithdrawAmountFiatUserCurrency.gt(minimumRuneWithdrawAmountFiatUserCurrency)
```

Comment:
suggestion: I took a little bit of time to understand this condition and why we need the latest part, maybe we could put this in a `isPoolAssetWithdrawAmountTooSmall` or something similar?

Reply:
added some comments for clarity: 69b9605579939a0ae20d77a1bbde96a628de56e9